### PR TITLE
Issue#26 map icons

### LINKED ISSUE #26: [Search Assistance] Map Icon, Map Interface
## User Story
As a **User**, I would like **to have a page that is a map with pins on it representing Chefs nearby** so that **I can search for Dishes and Chefs based on location**.

## Detailed Description
User can search for chefs on map. Clicking on a pin will cause a card with chef details to show up at bottom of screen. Clicking the card will direct user to the chef's page.

## Acceptance Criteria
- [x] Given **that a user clicks on the map icon while on the search page**, then **they will be redirected to the map page that is centered in on their location **
- [x] Given **that a user clicks on a pin**, then **a chef card appears at the bottom of the screen which redirects to chef's page when clicked**
- [x] Given **that a user clicks on another part of the map that is not a pin*, then **the card will disappear if it was on the screen**

### LINKED ISSUE #27: [Search Assistance] Map Pin Interface
## User Story
As a **User**, I would like **to be able to click on a pin in the map and see the Chef that it corresponds with as well as their currently available dishes** so that **move forward with ordering after I find someone from the map functionality**.

## Detailed Description
Provide additional details and context.

## Acceptance Criteria
- [x] Given **User is on map page**, then **Chef location pins are shown**
- [x] Given **User clicks on pin**, then **Chef corresponding to pin is shown**
- [x] Given **Chef pin has been clicked**, then **Chef's available dishes are shown**

### PR DESCRIPTION
**Pull request by nsrikumar1**
Trying to resolve merge issues

### CODE REVIEW

**Comment by nsrikumar1** [comment]
Uggh, idk why it is not passing the checks, will figure out later, look through if you see reason why

File: app/screens/Search.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ function Search(props){
 
     return(
         <SafeAreaProvider style={styles.container}>
-            <NavBarComponent search={true} navigation={props.navigation}/>
+            <NavBarComponent search="true" navigation={props.navigation}/>
```
Comment (shikham-8) [inline app/screens/Search.js:93]: Does this not need the { } around the prop?

**Comment by nsrikumar1** [comment]
image
image

File: app/screens/Search.js (hunk 1 above)
Comment (nsrikumar1) [reply app/screens/Search.js:93]: You don't need for strings or ints, I believe. I changed it on accident, but this still works.

**Comment by nsrikumar1** [comment]
When you click on a pin, it centers the map on the pin and displays a chef card, which directs to the chef when clicked on

**Review by jcgubbi — APPROVED** [review]
looks great!

**Pull request merged**